index.html

```diff
@@ -1799,39 +1814,40 @@ <h3>Securing Verifiable Credentials</h3>
 
         <p>
 At least one securing mechanism, and the details necessary to evaluate it, MUST
-be expressed for a <a>credential</a> or <a>presentation</a> to be a
-<a>verifiable credential</a> or <a>verifiable presentation</a>; that is, to be
-<a>verifiable</a>.
+be expressed to construct a <a>verifiable credential</a> or a
+<a>verifiable presentation</a>.
```

Comment:
The above text doesn't really make sense. Expressed where and how? It just says that someone must say that a securing mechanism exists somewhere in the wild ... and if they do that, then someone can construct a VC.

Perhaps if we say this instead it will have more force and (I hope) we can get to consensus:

"It MUST be possible to test whether a <a>verifiable credential</a> or <a>verifiable presentation</a> has been secured by at least one securing mechanism."

Reply:
Fixed in 073c722ae6d0f75113f89afec809de3874ee281e.